docker-images/zrok-private-access/docker-compose.yml

```diff
@@ -0,0 +1,64 @@
+version: '3'
+services:
+  zrok-enable-init:
+    image: busybox
+    # matches uid:gid of "nobody" in zrok container image
+    command: chown -Rc 65534:65534 /mnt/.zrok
+    user: root
+    volumes:
+      - zrok_env:/mnt/.zrok
+  zrok-enable:
+    image: docker.io/openziti/zrok:enable-headless  # FIXME: resume :latest if >= :0.3.2
```

Comment:
what's the fixme for?

Reply:
temp image tag has needed feature `--headless` not yet released